README.md

```diff
@@ -21,3 +21,47 @@ test('can be clicked', function(done) {
   MockInteractions.tap(button);
 });
 ```
+
+## Mocha Addons
+
+The Iron Mocha Addons are helpers that enable easier application of certain
+testing strategies for your elements. The addons automatically reverse their
+effects after each test.
+
+The following addons are implemented:
+
+### `stub`
+
+The `stub` addon allows the tester to partially replace the implementation of an
+element with some custom implementation. Usage example:
+
+```javascript
+beforeEach(function() {
+  stub('x-foo', {
+    attached: function() {
+      // Custom implementation of the `attached` method of element `x-foo`..
+    },
+    otherMethod: function() {
+      // More custom implementation..
+    }
+    // etc..
+  });
+});
+```
+
+### `replace`
+
+The `replace` addon allows the tester to replace all usages of one element with
+another element within all Polymer elements created within the time span of the
+test. Usage example:
+
+```javascript
+beforeEach(function() {
+  replace('x-foo').with('x-fake-foo');
```

Comment:
`stub` above replaces "just the implementation." There are cases where you do not care about styling as much as you care about replacing a complex element implementation. Between the two helpers, you'll probably get 95% of the way there.